lib/ansible/modules/cloud/kubevirt/kubevirt_raw.py

```diff
@@ -195,6 +195,21 @@
             description: Current status details for the resource.
             returned: success
             type: complex
+        wait:
+           description: Set to 'true' or 'yes' to block on the action
+                        till it completes.
+           returned: success
+           type: bool
+        timeout:
+           description: Specifies how much time in seconds to wait for an
+                        action to compelte if 'wait' option is enabled.
+           returned: success
+           type: int
+        interval:
+           description: Specifies how the interval in seconds between polling
+                        of the watched entity
```

Comment:
"Specifies the length of the interval in seconds between polling the watched entity" or maybe "Specifies the frequency of polling the watched entity in seconds"